classes/field/util.php

```diff
@@ -388,7 +388,7 @@ public static function get_default( $field, array  $form, $convert_opt = false )
 			$default = false;
 		}
 
-		if( $convert_opt && 0 === strpos( $default, 'opt' ) ){
+		if( is_array( $default ) || ( $convert_opt && 0 === strpos( $default, 'opt' ) ) ){
```

Comment:
I am testing with a checkbox field with no default saved and the filter described here: https://github.com/CalderaWP/Caldera-Forms/pull/2322#issuecomment-368626113

I get this error
`Warning: strpos() expects parameter 1 to be string, array given in /app/wp-content/plugins/caldera-forms/classes/field/util.php on line 391`

BTW That makes 0% sense. I checked with xdebug that $default is an array, so you would think `strpos()` would not be executed, but it is. PHP is a troll sometimes.

Reply:
There is only one form in my field, I watched every time it was executed. I'm going to stop arguing with the compiler and just fix this.